Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Interop.WPF/WindowsXamlHost.cs

```diff
@@ -0,0 +1,112 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Windows;
+using System.Windows.Interop;
+
+namespace Microsoft.Toolkit.Win32.UI.Interop.WPF
+{
+    /// <summary>
+    /// WindowsXamlHost control hosts UWP XAML content inside the Windows Presentation Foundation
+    /// </summary>
+    public class WindowsXamlHost : WindowsXamlHostBase
```

Comment:
It's not clear to me when we would use `WindowsXamlHostBase` vs `WindowsXamlHostBaseExt`

Reply:
I created the Ext sub-class since I needed some of what was in WindowsXamlHost, but didn't want all of that.  The only wrapper examples that could use WindowsXamlHostBase were the relatively simple Rectangle and Button examples.